src/pages/ThorChainLP/components/ReusableLpStatus/TransactionRow.tsx

```diff
@@ -86,10 +89,20 @@ export const TransactionRow: React.FC<TransactionRowProps> = ({
   const [txFeeCryptoPrecision, setTxFeeCryptoPrecision] = useState<string | undefined>()
 
   const { currentAccountIdByChainId, positionStatus } = confirmedQuote
+  const { type } = fromQuote(confirmedQuote)
 
   const asset = useAppSelector(state => selectAssetById(state, assetId))
+
+  const isAssetWithdraw =
+    isLpConfirmedWithdrawalQuote(confirmedQuote) && opportunityType === 'asset' && isWithdraw
+
+  const isRuneTx = useMemo(
+    () => assetId === thorchainAssetId || isAssetWithdraw,
+    [assetId, isAssetWithdraw],
+  )
+
   const feeAsset = useAppSelector(state =>
-    selectFeeAssetByChainId(state, fromAssetId(assetId).chainId),
+    selectFeeAssetByChainId(state, fromAssetId(isRuneTx ? thorchainAssetId : assetId).chainId),
```

Comment:
If the manually selected withdraw type is asym `asset` but the current opportunity type is `sym`, we should use `RUNE` as fee asset and send a tx using thorchain

Reply:
Do we want to add that comment as a comment?

Reply:
I don't think, this file as plenty of isRuneTx conditions that might require comments if so 👀 

But as you wish

Reply:
this also doesn't seem to be necessary. `assetId` should determine the state of the transaction which is set appropriately in `ReusableLpStatus`